front-end/src/components/analysis-page/AnalysisPage.jsx

```diff
@@ -3,38 +3,78 @@ import { connect } from "react-redux";
-                    cursor-pointer m-5 w-[5em]"
-                  role="button"
-                  onClick={this.openMainPage}
-                  aria-label="Home Button"
-                  tooltip="Click here to return to homepage"
-                  data-testid="logo-button"
-                >
-                  <Logo />
-                </div>
-                <div className="w-4/5 h-4/5 mx-auto">
-                  <IntentLister intents={analyzedTranscripts[projectId]} />
-                </div>
+      return (
+        <div className="AnalysisPage" data-testid="analysis-page">
+          <div /* display sidebar if toggled on */>
+            <IntentMenu
+              intents={analyzedTranscripts[projectId]}
+              isOpen={this.state.showIntentMenu}
+              onClickOutside={this.hideIntentMenu}
+            />
+          </div>
+          <div /* darken screen if sidebar is toggled on */
+            className={"bg-purple-100 h-screen transition ease-in-out" + (this.state.showIntentMenu ? " brightness-75" : "")}
+            data-testid="analysis-page-main"
+          >
+            <div className="flex gap-y-10 w-full flex-col h-full">
+              <div /* second line of className is tooltip styling */
+                className="justify-left relative text-center text-sm
+                  before:z-10 before:absolute before:-right-5 before:top-1/2 before:w-max before:max-w-xs before:translate-x-full before:-translate-y-1/2 before:rounded-md before:bg-gray-200 before:px-3 before:py-2 before:text-off-white before:invisible before:content-[attr(tooltip)] after:z-10 after:absolute after:-right-[0.8rem] after:top-1/2 after:h-0 after:w-0 after:translate-x-2 after:-translate-y-1/2 after:border-8 after:border-r-gray-200 after:border-l-transparent after:border-b-transparent after:border-t-transparent after:invisible hover:before:visible hover:after:visible
+                  cursor-pointer m-5 w-[5em]"
+                role="button"
+                onClick={this.openMainPage}
+                aria-label="Home Button"
+                tooltip="Click here to return to homepage"
+                data-testid="logo-button"
+              >
+                <Logo />
               </div>
+              <div className="w-4/5 h-4/5 mx-auto">
+                <IntentLister intents={analyzedTranscripts[projectId]} />
+              </div>
+            </div>
+
+            <div /*TEMPORARY SIDEBAR CONTROL BUTTON - WHEN DELETING THIS, DELETE BASEBUTTON IMPORT TOO!*/
+              className="fixed right-40 bottom-8" data-testid="arrow-right"
+            >
+              <BaseButton
+                click={this.toggleIntentMenu}
+                icon={{
+                    name: "hamburger",
+                    size: "40"
+                }}
+              />
```

Comment:
What is the plan to replace this

Reply:
Putting a hamburger icon button on the far left of the navbar (which is on Kaylee's branch and not this one).